src/main/java/org/folio/roles/integration/kafka/model/CapabilityEvent.java

```diff
@@ -30,6 +32,11 @@ public class CapabilityEvent {
    */
   private List<FolioResource> resources;
 
+  /**
+   * Permission data mapping overrides.
+   */
+  private Map<String, PermissionData> permissionMappingOverrides;
```

Comment:
Shouldn't this be ignored by Json deserialization or it's not an object being deserialized?

Reply:
added JsonIgnore